test/integration/golden_response/staging/get_stat_set/2010.json

```diff
@@ -6,24 +6,26 @@
           "date": "2010",
           "value": 36760,
           "metadata": {
-            "importName": "WorldDevelopmentIndicators",
-            "provenanceUrl": "https://datacatalog.worldbank.org/dataset/world-development-indicators/",
-            "observationPeriod": "P1Y",
-            "unit": "InternationalDollar"
+            "importName": "WorldDevelopmentIndicators"
```

Comment:
this change to a public api might be problematic. can we maintain backwards compatibility for these?

Reply:
These two APIs are not used since implemented (both internally or externally)  ![image](https://user-images.githubusercontent.com/5951856/117079800-5e0df180-acf1-11eb-8c4c-e464cef2c3bb.png)

Should we use them for a while, gather feedback for interface and then publicly release them (with /v1/ prefix to the REST path as well). I feel this is an alpha version with a high chance of more changes..

Reply:
Also hope to get all the APIs in [design](https://docs.google.com/document/d/12rdNsaLsMj5LxUWS3dynC8fuYTd0sfxqHio0Hh8Joao/edit#heading=h.9ssx4uezxkkc) implemented and used. When we feel confident about the interface, can release all of them as /v1/stat/*** at the same time

Reply:
that sounds like plan. nice to be able to make the change now while we can.